product/opni/components/Clusters.vue

```diff
@@ -1,28 +1,36 @@
 <script>
 import SortableTable from '@/components/SortableTable';
-import { getClusters } from '@/product/opni/utils/requests';
+import { getClusters, getClusterStats } from '@/product/opni/utils/requests';
 import Loading from '@/components/Loading';
-import AddTokenDialog from './dialogs/AddTokenDialog';
+import EditClusterDialog from './dialogs/EditClusterDialog';
 
 export default {
   components: {
-    AddTokenDialog, Loading, SortableTable
+    EditClusterDialog, Loading, SortableTable
   },
   async fetch() {
     await this.load();
+    await this.loadStats();
   },
 
   data() {
     return {
-      loading:  false,
-      clusters:  [],
-      headers:  [
+      loading:       false,
+      statsInterval: null,
+      clusters:      [],
+      headers:       [
         {
-          name:      'nameDisplay',
-          labelKey:  'tableHeaders.name',
-          sort:      ['nameDisplay'],
-          value:     'nameDisplay',
-          width:     undefined
+          name:          'nameDisplay',
+          labelKey:      'tableHeaders.name',
+          sort:          ['nameDisplay'],
+          value:         'nameDisplay',
+          width:         undefined,
+          formatter:     'Monospace',
+          formatterOpts: {
+            condition: (text) => {
+              return /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i.test(text);
```

Comment:
With the regexes you have here and other places can you add a comment with examples so it's easier to figure out what the intended match is.